main/unbound/unbound.initd

```diff
@@ -10,10 +10,13 @@ description="unbound is a Domain Name Server (DNS) that is used to resolve host
 description_configtest="Run syntax tests for configuration files only."
 description_reload="Kills all children and reloads the configuration."
 
-
 UNBOUND_BINARY=${UNBOUND_BINARY:-/usr/sbin/unbound}
 UNBOUND_CHECKCONF=${UNBOUND_CHECKCONF:-/usr/sbin/unbound-checkconf}
-UNBOUND_CONFFILE=${UNBOUND_CONFFILE:-/etc/unbound/${SVCNAME}.conf}
+UNBOUND_CONFFILE=${UNBOUND_CONFFILE:-/etc/unbound/${RC_SVCNAME}.conf}
+
+command=$UNBOUND_BINARY
```

Comment:
Please remove this variable and add `/usr/sbin/unbound` directly. We really don’t expect, nor support, that users will move installed binaries. ;)